pom.xml

```diff
@@ -32,6 +32,10 @@
 			<groupId>se.sundsvall.dept44</groupId>
 			<artifactId>dept44-common-validators</artifactId>
```

Comment:
Remove schedlock dependencies and version property? This is included in dept44-starter-scheduler, right?